crowbar_framework/config/initializers/sqlite_transaction_monkey_patch.rb

```diff
@@ -24,7 +24,14 @@ module ActiveRecord
   module ConnectionAdapters
     class SQLite3Adapter < AbstractAdapter
       def begin_db_transaction
-        log("begin transaction",nil) { @connection.transaction(:immediate) }
+        5.times do
+          begin
+            log("begin transaction",nil) { @connection.transaction(:immediate) }
+            break
+          rescue SQLite3::BusyException => e
+            log("begin transaction fails with #{e.to_s}", nil)
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (https://github.com/bbatsov/ruby-style-guide#no-to-s)